components/ArrowLeft/index.js

```diff
@@ -1,10 +1,13 @@
 import Link from "next/link";
 import { StyledArrow } from "./ArrowLeft.styles";
+import { FaAngleLeft } from "react-icons/fa6";
 
 export default function ArrowLeft() {
   return (
     <StyledArrow>
-      <Link href="../">⬅️</Link>
+      <Link href="../">
```

Comment:
If you want to send the user back to the homepage then i think it would be better to do the following. 

```suggestion
      <Link href="/">
```